services/QuillLMS/app/controllers/api/v1/users_controller.rb

```diff
@@ -3,25 +3,7 @@
 class Api::V1::UsersController < Api::ApiController
 
   def index
-    has_refresh_token = false
-    refresh_token_expires_at = nil
-    if current_user
-      user_id = current_user.id
-      auth_credential = AuthCredential.where(user_id: user_id).first
-      if auth_credential.present?
-        if auth_credential.refresh_token
-          has_refresh_token = true
-          refresh_token_expires_at = auth_credential&.expires_at&.in_time_zone&.utc&.to_s&.sub(' ','T')
-        end
-      end
-    end
-
-    render json: {
-      user: current_user,
-      text: "Hi",
-      has_refresh_token: has_refresh_token,
-      refresh_token_expires_at: refresh_token_expires_at
-    }
+    render json: { user: current_user }
```

Comment:
Yeah, it looks like you removed the usage of the refresh token fields on the frontend in [this PR in July](https://github.com/empirical-org/Empirical-Core/pull/7851/files#diff-3cc3eb04a48dcab4cc4fd7b4aa44b4c9c831272f251878399631be49eeca036e) when removing the google re-auth code. So if that's no longer needed, this change seems good. 

Side note, I'm assuming/hoping the `text: "Hi!` is also vestigial (likely test) code. It was put in place in the initial controller code [in 2017](https://github.com/empirical-org/Empirical-Core/commit/d642a95212ba64b52d7faab1b75f1d91984fa11d) and kept being carried forward. I can't find references to it, but `text` is generic enough to be hard to search for.

Reply:
I wasn't able to find anything and the requests to that API don't seem to use it.